Foods.yaml

```
@@ -6,4 +6,5 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara"]
+
+  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara", "Lasagna]
```

It was pointed out to me yesterday that we should try to stick with `snake_case` for `yaml` files
```suggestion
  - ["pizza", "kale_smoothies", "cheese", "pie", "carbonara", "lasagna]
```